ibraheemosule/skoolizy
# za - updated folder names

# Why this PR?
This PR aims to rename folders to enhance code readability and ensure consistent, easy access to critical components

# What does this PR do?
- Renames folders for clarity and consistency.
- Updates file paths and imports to reflect these changes.

# How to test your work?
Ensure that components load as expected in their respective views and functionality remains intact.

# Pages to see your changes?
/media
/announcements
